examples/tinygo-outbound-mysql/spin.toml

```diff
@@ -13,6 +13,7 @@ component = "tinygo-mysql"
 [component.tinygo-mysql]
 environment = { DB_URL = "mysql://spin:spin@127.0.0.1/spin_dev" }
 source = "main.wasm"
+allowed_outbound_hosts = ["mysql://127.0.0.1:3306"]
```

Comment:
fwiw `:3306` should be the default for `mysql://`

Reply:
We currently require the port even if there is a scheme. Changing this is super easy, but I wanted to start with the most conservative approach and expand it if/when we think it's sensible.

Reply:
I thought `allowed_outbound_hosts` entries were meant to be just host + port rather than scheme?  Is scheme allowed but ignored?

Reply:
(`allowed_http_hosts` allowed the HTTP/S scheme for backward compatibility because there was already existing code that used it, but we tried to remove the scheme from the docs in favour of "just the host".)

Reply:
#1988 makes default ports optional

Reply:
To expand on what @lann said: scheme is always required and the port can omitted when they scheme has a default well known port. Since MySQL has well known default port (i.e., 3306), the port can be omitted.

Reply:
Thanks all; updated.